app/scripts/HiGlassComponent.js

```diff
@@ -2461,7 +2479,11 @@ class HiGlassComponent extends React.Component {
     totalTrackHeight += MARGIN_HEIGHT;
     const rowHeight = this.state.rowHeight + MARGIN_HEIGHT;
 
-    if (!this.props.options.bounded) {
+    if (this.props.options.scrollable) {
```

Comment:
Comment explaining why this is necessary would be nice.

Reply:
The difference boils down to the fact that normally `totalTrackHeight` is the same as the container height but when the user enables scrolling mode `totalTrackHeight` could be larger or smaller than the container.

I'll add a comment